testsuite/forge/src/backend/local/mod.rs

```diff
@@ -126,6 +128,11 @@ impl LocalFactory {
     where
         R: ::rand::RngCore + ::rand::CryptoRng,
     {
+        let swarmdir = if self.swarm_dir.is_empty() {
+            None
+        } else {
+            Some(PathBuf::from(self.swarm_dir.as_str()))
```

Comment:
nit: could you use `map` to avoid this? https://doc.rust-lang.org/std/option/enum.Option.html#method.map